pkg/server/endpoints/registration/handler.go

```diff
@@ -536,6 +536,46 @@ func (h *Handler) FetchBundle(
 	}, nil
 }
 
+//Evict removes (de-attest) node from attested nodes store
+func (h *Handler) EvictAgent(ctx context.Context, evictRequest *registration.EvictAgentRequest) (*registration.EvictAgentResponse, error) {
+	spiffeID := evictRequest.GetSpiffeID()
+	err := h.deleteAttestationEntry(ctx, spiffeID)
+	if err != nil {
+		h.Log.Warnf("Fail to de-attested agent with ID: %q", spiffeID)
+		return &registration.EvictAgentResponse{
+			DeleteSucceed: false,
+		}, err
+	}
+
+	h.Log.Infof("Successfully de-attested agent with ID: %q", spiffeID)
+	return &registration.EvictAgentResponse{
+		DeleteSucceed: true,
+	}, nil
+}
+
+//List returns the list of attested nodes
+func (h *Handler) ListAgents(ctx context.Context, empty *common.Empty) (*registration.ListAgentsResponse, error) {
+	dataStore := h.Catalog.DataStores()[0]
+	listAttestedNodesRequest := &datastore.ListAttestedNodesRequest{}
+	listAttestedNodesResponse, err := dataStore.ListAttestedNodes(ctx, listAttestedNodesRequest)
```

Comment:
This could be a little more succinct and easier to read if the variable names were simply `req` and `resp`... the variables are scoped to this function, the golang idiom is shorter names are better particularly if they are declared and used very close together